backbone.queryparams.js

```diff
@@ -360,7 +360,11 @@ function iterateQueryString(queryString, callback) {
 }
 
 function containsInvalidKey(keys) {
-  return INVALID_KEYS.some(invalidKey => keys.includes(invalidKey));
+  // Square brackets will be removed down the line so
+  // we want to make sure they are filtered upfront.
+  const regex = /(\[|\])+/gi;
+  var sanitizedKeys = keys.replace(regex, '');
+  return INVALID_KEYS.some(invalidKey => sanitizedKeys.includes(invalidKey));
```

Comment:
Does it matter if one of the key from URL is something like `theprototypes`? I don't think so. But I might not have understood this fully.
If it does not matter, this line should be simplified to something like `return INVALID_KEYS.includes(sanitizedKey.toLowerCase());`

Reply:
This is to help against filter evasion (https://owasp.org/www-community/xss-filter-evasion-cheatsheet).